src/datasets/other/serp_events/reference.md

```diff
@@ -430,73 +432,108 @@ Number of SERP impressions with ads loaded and an ad blocker in use:
+| `is_private`                                 | Was the SERP loaded while in Private Browsing Mode?                                                                                                                                                                                                          |
+| `is_signed_in`                               | Was the SERP loaded while signed into a search provider account?                                                                                                                                                                                             |
+| `search_engine`                              | `google`, `bing`, `duckduckgo`, `ecosia` (only these support SERP events currently).                                                                                                                                                                         |
+| `sap_source`                                 | How the user arrived at the SERP [e.g. `urlbar`, `follow_on_from_refine_on_SERP`]. There are a number of cases where this will be `unknown`, e.g. clicking on a link that opens a new SERP, or clicking on a history result containing a SERP URL.           |
+| `is_tagged`                                  | Whether the search is tagged (`true`) or organic (`false`).                                                                                                                                                                                                  |
+| `abandon_reason`                             | Why the SERP is deemed abandoned: `tab_close`, `window_close`, `navigation`, or `null` if not abandoned.                                                                                                                                                     |
+| `ad_components`                              | Array with 1 entry for each ad component which had either an impression or engagement, `[]` if none.                                                                                                                                                         |
+| `ad_components.component`                    | SERP display component containing ad links [e.g. `ad_link`, `ad_carousel`].                                                                                                                                                                                  |
+| `ad_components.num_loaded`                   | Number of ads loaded in the component. They may or may not be visible on the page.                                                                                                                                                                           |
```

Comment:
Maybe clarify that they may or may not be visible due to ad blocker or the behavior of the component